test/unit/mock/command-executor-mock.js

```diff
@@ -0,0 +1,6 @@
+class CommandExecutorMock {
+    /* eslint-disable-next-line no-unused-vars */
+    add(addCommand) {}
```

Comment:
Please avoid using eslint disable, if we don't have default behaviour for the method, you can remove it from mock definition